### PR TITLE
Fix "HumanReadable" function name.

### PR DESCRIPTION
**Pull request by lavalamp**

### CODE REVIEW

File: fieldpath/element.go [hunk 1]
```diff
@@ -55,14 +55,14 @@ func (e PathElement) String() string {
 	case len(e.Key) > 0:
 		strs := make([]string, len(e.Key))
 		for i, k := range e.Key {
-			strs[i] = fmt.Sprintf("%v=%v", k.Name, k.Value.HumanReadable())
+			strs[i] = fmt.Sprintf("%v=%v", k.Name, k.Value.String())
```
Comment (apelisse) [inline fieldpath/element.go]: Isn't `String()` automatically called when you use `%v`? If so you can drop it everywhere

Comment (lavalamp) [reply fieldpath/element.go]: Yes, hazards of using a rename tool :) will fix

**Comment by lavalamp** [comment]
omitted String() calls.

**Pull request merged**